### DISCUSSION TITLE
Its possible to support 3 Huawei M1 inverters and 2x Luna 2000 S0 ?

### DISCUSSION THREAD

**Discussion by ottosax** [JensenNick/huawei_solar_pees#90] [Forum]
Hi,
I have a setup with 2x Huawei 10 KTL M1 and one 8 KTL M1 inverters cascaded with an sDongle at the master inverter, backup box, and 2x Luna 2000 S0 (15 and 10 kWh) cascaded to the master inverter (10 KTL M1).
Its possible to use and configure it with huawei_solar_pees?
![20230305_184521](https://github.com/user-attachments/assets/5aa8c8ae-c97e-46ae-9290-e84a04e89759)

Greetings

**Comment by JensenNick** [comment]
Possible yes, but if you’re asking me to implement it, I’m afraid I have to disappoint you, at least for now. I would not be able to run any tests / check so it would require someone to commit to help with the setup you mention and right now I have other projects which I prioritise. I'm adding your request to the roadmap.

**Comment by ottosax** [comment]
OK, that's a shame - but if you ever find the time, I'd be happy to volunteer as a tester. A little basic IT knowledge would be required. :-)
